<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>323</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>667</width>
    <height>323</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>667</width>
    <height>323</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Scrcpy Camera Controller</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="MainPage">
       <attribute name="title">
        <string>Main</string>
       </attribute>
       <widget class="QLabel" name="serialLabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>221</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Enter your device serial number - </string>
        </property>
       </widget>
       <widget class="QLineEdit" name="serialNumber">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>50</y>
          <width>171</width>
          <height>32</height>
         </rect>
        </property>
       </widget>
       <widget class="QCheckBox" name="debugEnabled">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>281</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Debug Window (shows terminal output)</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="previewEnabled">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>130</y>
          <width>271</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Enable preview window</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="paramsEnabled">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>160</y>
          <width>261</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Addtional Parameters</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="paramsToEnable">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>190</y>
          <width>171</width>
          <height>32</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="statusLabel">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>10</y>
          <width>58</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Status - </string>
        </property>
       </widget>
       <widget class="QLabel" name="batyLabel">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>40</y>
          <width>58</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Battery - </string>
        </property>
       </widget>
       <widget class="QProgressBar" name="batyPercentage">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>40</y>
          <width>161</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
       <widget class="QLabel" name="deviceConnectionStatus">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>80</y>
          <width>211</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>this is a status message</string>
        </property>
       </widget>
       <widget class="QLabel" name="operatinglabel">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>110</y>
          <width>101</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Operating at - </string>
        </property>
       </widget>
       <widget class="QLabel" name="operationStatus">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>140</y>
          <width>191</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>lalalalalaliiilalala at lallalala Hz</string>
        </property>
       </widget>
       <widget class="QPushButton" name="connectionButton">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>180</y>
          <width>88</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="verboseEnabled">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>180</y>
          <width>131</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Enable Logging</string>
        </property>
       </widget>
       <widget class="QLabel" name="loggingMessageIfEnabled">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>221</y>
          <width>651</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>7</pointsize>
         </font>
        </property>
        <property name="text">
         <string>!! Logging enabled. The program will output verbose data on a log file created within it's directory. Log output will happen when errors happen anyways.</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="SettingsPage">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <widget class="QLabel" name="rezLabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>251</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Device Resolution (defaults to 640x480) - </string>
        </property>
       </widget>
       <widget class="QLabel" name="widthLabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>58</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Width - </string>
        </property>
       </widget>
       <widget class="QLabel" name="heightLabel">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>47</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Height -   </string>
        </property>
       </widget>
       <widget class="QLineEdit" name="rezWidth">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>40</y>
          <width>71</width>
          <height>32</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="rezHeight">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>40</y>
          <width>71</width>
          <height>32</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="frameRateLabel1">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>221</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Device Frame Rate (defauts to 30) - </string>
        </property>
       </widget>
       <widget class="QLabel" name="frameRateLabel2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>71</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Frame Rate = </string>
        </property>
       </widget>
       <widget class="QLineEdit" name="deviceFrameRate">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>100</y>
          <width>113</width>
          <height>32</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="bitrateLabel1">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>140</y>
          <width>541</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Device BitRate (Default = 8)</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="deviceBitRate">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>170</y>
          <width>113</width>
          <height>32</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="bitrateLabel2">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>170</y>
          <width>211</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>MegaBits per second </string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="helpOutput">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>40</y>
          <width>281</width>
          <height>131</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="helpOutputLabel">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>10</y>
          <width>251</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Detected ADB Serials and Camera IDs -  </string>
        </property>
       </widget>
       <widget class="QLabel" name="cameraIDLabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>210</y>
          <width>261</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Camera ID (what camera you are using) - </string>
        </property>
       </widget>
       <widget class="QLineEdit" name="deviceID">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>200</y>
          <width>113</width>
          <height>32</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="updateHelpOutput">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>190</y>
          <width>121</width>
          <height>34</height>
         </rect>
        </property>
        <property name="text">
         <string>Update Output</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="About">
       <attribute name="title">
        <string>About</string>
       </attribute>
       <widget class="QLabel" name="signatureLabel">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>220</y>
          <width>461</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Scrcpy GUI Camera Controller. Made with &lt;3 by Rudraksh Tiku (RudyDaBot)</string>
        </property>
       </widget>
       <widget class="QLabel" name="controllerVersion">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>461</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Version = whatever the fuck it is, code's gonna control this text anyways...</string>
        </property>
       </widget>
       <widget class="QLabel" name="scrcpyVersion">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>181</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>scrcpy version - lalalalallala</string>
        </property>
       </widget>
       <widget class="QLabel" name="ADBVersion">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>231</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>ADB version = shimmy shimmy yay</string>
        </property>
       </widget>
       <widget class="QLabel" name="kanavEasterEgg">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>100</y>
          <width>411</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Kanav if you're reading this, you a bitch.</string>
        </property>
       </widget>
       <widget class="QPushButton" name="enableEgg">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>210</y>
          <width>88</width>
          <height>34</height>
         </rect>
        </property>
        <property name="text">
         <string>Easter Egg</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
